modules/App.js

```diff
@@ -1,12 +1,10 @@
-/** @jsx React.DOM */
+import React from 'react';
+import {Link, RouteHandler} from 'react-router'
 
-var React = require('react');
-var {Link, RouteHandler} = require('react-router');
+import './App.less';
 
-require('./App.css');
-
-var App = React.createClass({
-  render: function() {
+class App extends React.Component {
```

Comment:
Pete and I are still using [React.createClasss)[https://facebook.github.io/react/docs/top-level-api.html]. Until you enable es7 features, it's sort of a hassle to use ES6 classes.